python/cac_tripplanner/templates/cac_tripplanner/cac_tripplanner_home.html

```diff
@@ -0,0 +1,79 @@
+{% extends "cac_tripplanner/base_cac_tripplanner.html" %}
```

Comment:
I was trying to stay close to the two-scoops style - but you're right, this application probably doesn't warrant such heavy conventions.